src/main/java/seedu/address/ui/CookbookCard.java

```diff
@@ -16,9 +16,9 @@
 /**
  * An UI component that displays information of a {@code Recipe}.
  */
-public class RecipeCard extends UiPart<Region> {
+public class CookbookCard extends UiPart<Region> {
```

Comment:
Following the same line of thought, we can also keep this as the original `RecipeCard`.

Reply:
Updated